src/controllers/music/musicartists.js

```diff
@@ -76,7 +83,9 @@ define(["layoutManager", "loading", "events", "libraryBrowser", "imageLoader", "
                         return;
                     }
 
-                    query.StartIndex -= query.Limit;
+                    if (userSettings.libraryPageSize() > 0) {
+                        query.StartIndex += query.Limit;
```

Comment:
```suggestion
                        query.StartIndex -= query.Limit;
```
or ideally `query.StartIndex = Math.max(0, query.StartIndex - query.Limit)`